modules/gapi/test/cpu/gapi_operators_tests_fluid.cpp

```diff
@@ -21,23 +21,32 @@ INSTANTIATE_TEST_CASE_P(MathOperatorTestFluid, MathOperatorMatMatTest,
                                 Values(cv::Size(1280, 720),
                                        cv::Size(640, 480),
                                        cv::Size(128, 128)),
-                                Values(-1, CV_8U, CV_32F),
+                                Values(-1),
                                 Values(CORE_FLUID),
                                 Values(AbsExact().to_compare_obj()),
                                 Values( opPlusM, opMinusM, opDivM,
                                         opGreater, opLess, opGreaterEq, opLessEq, opEq, opNotEq)));
 
-//FIXME: Some Mat/Scalar Fluid kernels are not there yet!
-INSTANTIATE_TEST_CASE_P(DISABLED_MathOperatorTestFluid, MathOperatorMatScalarTest,
+INSTANTIATE_TEST_CASE_P(MathOperatorArithmeticTestFluid, MathOperatorMatScalarTest,
                         Combine(Values(CV_8UC1, CV_16SC1, CV_32FC1),
                                 Values(cv::Size(1280, 720),
                                        cv::Size(640, 480),
                                        cv::Size(128, 128)),
-                                Values(-1, CV_8U, CV_32F),
```

Comment:
Why

Reply:
This argument is for `dtype` in our `GAPI_TEST_FIXTURE`
And operators are physically not able to change the destination Mat type, so these tests parameters were just misleading.
I fixed it in all the operators' tests

Reply:
OK I see it now, thanks